src/views/projectExplorer/libraryList.ts

```diff
@@ -8,6 +8,9 @@ import { ContextValue } from "../../projectExplorerApi";
 import { ProjectManager } from "../../projectManager";
 import Library, { LibraryType } from "./library";
 import { getInstance } from "../../ibmi";
+import ErrorItem from "./errorItem";
+
+export type Position = 'first' | 'last' | 'middle';
```

Comment:
This type is defined in both this file and also `includePaths.ts`. I think we can remove both and have it once at the top of the `iproject.ts` file.